### DISCUSSION TITLE
Community Engagement Ideas

### DISCUSSION THREAD

**Discussion by tomvothecoder** [xCDAT/xcdat#401] [3. Ideas]
This thread is a way for us to brainstorm community engagement ideas and keep track of ones we decide to do.

## Ideas

To do:
- [x] Add a section to the README/documentation that cites other software packages/libraries that use xCDAT
- [ ] Project Pythia cookbooks
	- [ ] Collaborate with ESGF on some notebooks (use-cases and workflows)
- [ ] Community calls/office hours
	-  Open open up our second bi-weekly core team call of the month to anyone in the community
	-  Example from xarray: https://github.com/pydata/xarray/discussions/7564

To consider:
- [ ] xCDAT Showcase
  - Seminar/presentation 
  - Demonstrate use cases and implementations in other projects

## Seminars

Upcoming:
- [ ] Pangeo Showcase Seminar Series (TBD)
  - https://github.com/xCDAT/xcdat/discussions/399
  - https://pangeo.io/pangeo-showcase.html

To consider:
- PCMDI/NCAR Seminar Series
-  ESGF Seminar
	- ESGF Nimbus Integration (conda env)
- AGU 
- CMIP6 Hackathon

Past Items:
- [x] https://github.com/xCDAT/xcdat/issues/433
- [x] AMS 2023 (1/11/23)
- [x] CWSS Seminar (1/25/23)
- [x] xCDAT Developer Day
  - 4-6 hours, do a sprint and work on issues we care about
  - Target users: 
    - Idea of what they want to implement (e.g., Celine and Sam with their work)
    - Invite people who are interested in pushing code, mention at CSI, and also provide guidelines for specific issues (also vet what others are interested in working in)
    - Keep it open to everybody else
  - Figure out a date after pushing a new version
     - Consider European attendees and timezones
   - Experiment with fork and branches 
   - Maybe just add other devs to a new group with write access
   - We can add this to the [Community Engagement Ideas ](https://github.com/xCDAT/xcdat/discussions/401)post

**Comment by jypeter** [comment]
@tomvothecoder a reboot of **some (x)CDAT mailing list would be useful!**

Something for announcements, and for questions (simple questions that can't go to github, or a place for people afraid of using github or who don't want to spend too much time describing a problem). Anything to advertise/promote the use of tools developed at PCMDI and useful for the larger climate modelling community

Trouble is what could you use as a ML server? The [old lists](https://cdat.llnl.gov/contact.html) were never very active, and received some spam from time to time. And the trouble is that you could not **access (and more importantly search) archives**. Some previous IT people at PCMDI tried to import the archives into something that could be searched (and that would be allowed to run on an LLNL server), but without success

**Reply by jypeter** [reply]
Wow, I have just found a **link to the cdat-discussion list** on sourceforge (**not advertised** on the [CDAT contacts](https://cdat.llnl.gov/contact.html) page...) when cleaning my old bookmarks. If somebody want to read 20+ years of CDAT related discussions!

It seems I wrote the oldest archived mail \o/

https://sourceforge.net/p/cdat/mailman/cdat-discussion/
